sources/Interop/SetupApi/um/SetupAPI/SetupApi.Manual.cs

```diff
@@ -0,0 +1,1408 @@
+
+        public const int DICLASSPROP_INSTALLER = 0x00000001;
+
+        public const int DICLASSPROP_INTERFACE = 0x00000002;
+
+        public const int DMI_MASK = 0x00000001;
+
+        public const int DMI_BKCOLOR = 0x00000002;
+
+        public const int DMI_USERECT = 0x00000004;
+
+        public const uint DIGCDP_FLAG_BASIC = 0x00000001;
+
+        public const uint DIGCDP_FLAG_ADVANCED = 0x00000002;
+
+        public const uint DIGCDP_FLAG_REMOTE_BASIC = 0x00000003;
+
+        public const uint DIGCDP_FLAG_REMOTE_ADVANCED = 0x00000004;
+
+        public const int IDI_RESOURCEFIRST = 159;
+
+        public const int IDI_RESOURCE = 159;
+
+        public const int IDI_RESOURCELAST = 161;
+
+        public const int IDI_RESOURCEOVERLAYFIRST = 161;
+
+        public const int IDI_RESOURCEOVERLAYLAST = 161;
+
+        public const int IDI_CONFLICT = 161;
+
+        public const int IDI_CLASSICON_OVERLAYFIRST = 500;
+
+        public const int IDI_CLASSICON_OVERLAYLAST = 502;
+
+        public const int IDI_PROBLEM_OVL = 500;
+
+        public const int IDI_DISABLED_OVL = 501;
+
+        public const int IDI_FORCED_OVL = 502;
+
+        public const int SPWPT_SELECTDEVICE = 0x00000001;
+
+        public const int SPWP_USE_DEVINFO_DATA = 0x00000001;
+
+        public const uint SIGNERSCORE_UNKNOWN = 0xFF000000;
+
+        public const uint SIGNERSCORE_W9X_SUSPECT = 0xC0000000;
+
+        public const uint SIGNERSCORE_UNSIGNED = 0x80000000;
+
+        public const uint SIGNERSCORE_AUTHENTICODE = 0x0F000000;
+
+        public const uint SIGNERSCORE_WHQL = 0x0D000005;
+
+        public const uint SIGNERSCORE_UNCLASSIFIED = 0x0D000004;
+
+        public const uint SIGNERSCORE_INBOX = 0x0D000003;
+
+        public const uint SIGNERSCORE_LOGO_STANDARD = 0x0D000002;
+
+        public const uint SIGNERSCORE_LOGO_PREMIUM = 0x0D000001;
+
+        public const uint SIGNERSCORE_MASK = 0xFF000000;
+
+        public const uint SIGNERSCORE_SIGNED_MASK = 0xF0000000;
+
+        public const int DICUSTOMDEVPROP_MERGE_MULTISZ = 0x00000001;
+
+        public const int SCWMI_CLOBBER_SECURITY = 0x00000001;
+
+        public const int SP_MAX_MACHINENAME_LENGTH = Windows.MAX_PATH + 3;
+
+        public const int SP_ALTPLATFORM_FLAGS_VERSION_RANGE = 0x0001;
+
+        public const int SP_ALTPLATFORM_FLAGS_SUITE_MASK = 0x0002;
+
+        public const int DIRID_IOSUBSYS = DIRID_DRIVERS;
+
+        public const int DIRID_DEFAULT = DIRID_SYSTEM;
+
+        public const int FILEOP_RETRY = FILEOP_DOIT;
+
+        public const uint SPID_ACTIVE = SPINT_ACTIVE;
+
+        public const uint SPID_DEFAULT = SPINT_DEFAULT;
+
+        public const uint SPID_REMOVED = SPINT_REMOVED;
+
+        public const uint NDW_INSTALLFLAG_NEEDRESTART = DI_NEEDRESTART;
+
+        public const uint NDW_INSTALLFLAG_NEEDREBOOT = DI_NEEDREBOOT;
+
+        public const uint ERROR_EXPECTED_SECTION_NAME = Windows.APPLICATION_ERROR_MASK | Windows.ERROR_SEVERITY_ERROR | 0;
```

Comment:
Same for these, the `Windows.` has been using static'd away in the rest of the codebase

Reply:
Fixed.